addons/sys_server/fnc_getRadioId.sqf

```diff
@@ -21,10 +21,10 @@ _class = toLower _class;
 
 GVAR(radioIdMap) params ["_baseTypes","_radioIds"];
 
-private _mapIndex = _baseTypes find (toLower _class);
```

Comment:
Yes, If you look at the beginning of the file, class is already set to lower cases.